Developer-Guide.md

```diff
@@ -131,6 +134,53 @@ $ sudo sed -i -e 's/^# *\(enable_debug\).*=.*$/\1 = true/g' /usr/share/defaults/
 $ sudo sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 agent.log=debug"/g' /usr/share/defaults/kata-containers/configuration.toml
 ```
 
+### journald rate limiting
+
+Enabling [full debug](#enable-full-debug) results in the kata components generating 
+large amounts of logging, which by default is stored in the system log. Depending on
+your system configuration, it is possible that some events may be discarded by the
+system logging daemon. The following details how to determing this for `systemd-journald`,
+and offers possible workarounds and fixes.
+
+> **Note** The method of implementaiton may vary between Operating System installations.
+> You may need to ammend these instructions as necessary to your system implementaiton,
+> and consult with your system administrator for appropriate configuration.
+
+#### `systemd-journald` suppressing messages
+
+`systemd-journald` may be configured to rate limit the number of journal entries
+it stores. When messages are suppressed, it is noted in the logs. This can be checked
+for by looking for those notifications, such as:
+
+```sh
+$ sudo journalctl --since today | fgrep Suppressed
+Jun 29 14:51:17 mymachine systemd-journald[346]: Suppressed 4150 messages from /system.slice/docker.service
+```
+
+This message indicates that a number of log messages from the `docker.service` slice were
+suppressed. In such a case, you can expect to have incomplete logging information
+stored from the Kata Containers components
+
+#### Reducing `systemd-journald` rate limiting
```

Comment:
It might be better to say "Disabling" rather than "Reducing" in the title as that's what the section documents.

Reply:
Ah, good spot. I've renamed the title/toc, but left mention of disabling in the body.